pkg/api/mlflow/service/artifact/storage/local_test.go

```diff
@@ -7,8 +7,79 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config"
 )
 
+// TestGetArtifact for the local storage implementation.
+func TestGetArtifact_Ok(t *testing.T) {
+	// setup
+	defaultArtifactRoot := t.TempDir()
+	runArtifactRoot := "/run-artifact-root/"
+	fileName := "file.txt"
+	fileContent := "artifact content"
+
+	err := os.MkdirAll(filepath.Join(defaultArtifactRoot, runArtifactRoot), os.ModePerm)
+	assert.Nil(t, err)
+
+	f, err := os.Create(filepath.Join(defaultArtifactRoot, runArtifactRoot, fileName))
+	assert.Nil(t, err)
+	_, err = f.Write([]byte(fileContent))
+	assert.Nil(t, err)
+
+	// invoke
+	storage, err := NewLocal(nil)
+	assert.Nil(t, err)
+
+	file, err := storage.Get(filepath.Join(defaultArtifactRoot, runArtifactRoot), fileName)
+	assert.Nil(t, err)
+	defer func() {
+		file.Close()
+		os.Remove(f.Name())
+	}()
+
+	// verify
+	assert.NotNil(t, file)
+	readBuffer := make([]byte, 20)
+	ln, err := file.Read(readBuffer)
+	assert.Nil(t, err)
+	assert.Equal(t, fileContent, string(readBuffer[:ln]))
+}
+
+func TestGetArtifact_Error(t *testing.T) {
+	// setup
+	defaultArtifactRoot := t.TempDir()
+	runArtifactRoot := "/run-artifact-root/"
+	fileName := "file.txt"
+	fileContent := "artifact content"
+
+	err := os.MkdirAll(filepath.Join(defaultArtifactRoot, runArtifactRoot), os.ModePerm)
+	assert.Nil(t, err)
+
+	f, err := os.Create(filepath.Join(defaultArtifactRoot, runArtifactRoot, fileName))
+	assert.Nil(t, err)
+	_, err = f.Write([]byte(fileContent))
+	assert.Nil(t, err)
```

Comment:
why do you even bother creating a file and writing to it here???

Reply:
but you don't read it at all!
you are trying to read a file named "some-other-item" as this is a an _error_ test

Reply:
ah okay will remove